src/common/mod.rs

```diff
@@ -0,0 +1,61 @@
+//! Structs, traits, and utils to develop [component](https://docs.viam.com/components/)
```

Comment:
Some of these are actually natively-included general purpose drivers (gpio_motor, adxl, mpu), so maybe we should note those separately?